test/smoke-fails/atomic-contended/Makefile

```diff
@@ -0,0 +1,64 @@
+include ../../Makefile.defs
+
+TESTNAME=atomic-contended
+TESTSRC_ALL=atomic-contended.cpp
+
+ifeq ($(TARGETS),)
+TARGETS =--offload-arch=$(AOMP_GPU)
```

Comment:
Should this test be limited to certain archs? For example, how does this test behave on gfx906?

Reply:
Not sure TBH. (For now I've only tested with gfx90a.)